### PR TITLE
[FIX] remove reviewer checklist

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

remove reviewer checklist

part of https://github.com/inmanta/inmanta-core/issues/5005

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [ ] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

**Comment by LauraStordeur** [comment]
Why are we removing the reviewer checklist from the web-console? 😄

**Comment by Hugo-Inmanta** [comment]
We're removing it from all repos since it wasn't being used much in practice. But if it is actually being used for this repo I won't remove it here.

**Comment by LauraStordeur** [comment]
all good, we don't really use it ;)

File: changelogs/unreleased/5005-delete-reviewer-part-of-PR-checklist.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+description: Remove reviewer checklist from the PR template.
+issue-nr: 5005
+change-type: patch
```
Comment (arnaudsjs) [inline changelogs/unreleased/5005-delete-reviewer-part-of-PR-checklist.yml:3]:
```suggestion
issue-nr: 5005
issue-repo: inmanta-core
change-type: patch
```

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

**Review by arnaudsjs — APPROVED** [review]

**Pull request closed without merging**